tests/forms/test_fields.py

```diff
@@ -0,0 +1,33 @@
+
+import pytest
+
+import wtforms
+
+from dmutils.forms.fields import EmailField, StripWhitespaceStringField
+
+
+class EmailForm(wtforms.Form):
+    field = EmailField()
+
+
+class StripWhitespaceStringFieldForm(wtforms.Form):
+    field = StripWhitespaceStringField()
+
+
+@pytest.mark.parametrize('form, field_class', (
+    (EmailForm(), EmailField),
+    (StripWhitespaceStringFieldForm(), StripWhitespaceStringField),
+))
+def test_form_with_field(form, field_class):
+    assert isinstance(form.field, field_class)
```

Comment:
I'm not sure I understand what this is testing.

Reply:
To be honest I don't remember.

I will take the test out.